plugins/auth-backend/src/providers/saml/provider.ts

```diff
@@ -112,27 +113,33 @@ export class SamlAuthProvider implements AuthProviderRouteHandlers {
   }
 }
 
-type SAMLProviderOptions = {
-  entryPoint: string;
-  issuer: string;
-  path: string;
+type SAMLProviderOptions = SamlConfig & {
   tokenIssuer: TokenIssuer;
   appUrl: string;
 };
 
+type SignatureAlgorithm = 'sha1' | 'sha256' | 'sha512';
+
 export const createSamlProvider: AuthProviderFactory = ({
   providerId,
   globalConfig,
   config,
   tokenIssuer,
 }) => {
-  const url = new URL(globalConfig.baseUrl);
   const entryPoint = config.getString('entryPoint');
   const issuer = config.getString('issuer');
   const opts = {
+    callbackUrl: `${globalConfig.baseUrl}/${providerId}/handler/frame`,
```

Comment:
Hmmm. There's actually a fix going in here as well. I should call it out in the patch nodes.